Content panel

Morban15 opened the pull request:
### Issues

_¿Qué se está resolviendo?_
Tenemos que crear el content panel de la página Programs y esta debe tener
título, una imagen y descripción.

#23 

### Todos

_¿Qué queda por hacer?_
Crear, revisar y probar

### Áreas impactadas

_¿Qué páginas o componentes están afectados?_
Programs 

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

Morban15 committed:
Creando el archivo de HTML para content_panel

Hemos creado el archivo content_panel.html

Morban15 committed:
Creando el archivo de HTML para content_panel

Hemos creado el archivo content_panel.html

Morban15 committed:
Agregando el archivo de CSS para content_panel

Hemos creado el archivo de content_panel.css. Tambien removimos
una carpeta no necesaria.

Morban15 committed:
Agregando el archivo de CSS para content_panel

Hemos creado el archivo de content_panel.css. Tambien removimos
una carpeta no necesaria.

Morban15 committed:
Agregando la pagina en ingles y español y el CSS

Hemos agregado la el objeto panel en programs en ingles y español.
Ademas agregamos el content_panel.css a site.css.

Morban15 committed:
Agregando la pagina en ingles y español y el CSS

Hemos agregado la el objeto panel en programs en ingles y español.
Ademas agregamos el content_panel.css a site.css.

Morban15 committed:
WIP

Estamos haciendo algunos cambios

Morban15 committed:
WIP

Estamos haciendo algunos cambios

Morban15 committed:
Agregando h2 al link

Agregamos h2 al link en content_panel.html

Morban15 committed:
Agregando h2 al link

Agregamos h2 al link en content_panel.html

Morban15 committed:
Merge branch 'content_panel' of https://github.com/cosola-macile/macile.org into content_panel

Morban15 committed:
Content Panel terminado

Hemos hecho algunos cambios en los archivos de programs y content_panel
para terminar.

misscs commented on app/modules/content_panel/content_panel.css:
```diff
@@ -0,0 +1,26 @@
+img {
+  height: 100%;
+}
```
Esta regla es super general y va a crear conflictos con todas las otras imagenes. Queremos usar una clase y veo que has agregado una en el html 👍

misscs commented on app/modules/content_panel/content_panel.css:
```diff
@@ -0,0 +1,26 @@
+  margin: 0;
+}
+
+@media only screen and (max-width: 900px) {
```
🤞 Queremos trabajar de pequeño hasta grande. No te preocupas por cambiar esto ahora. Vamos a mejorarlo cuando agregamos RWD images

Morban15 replied:
Está bien

misscs commented on app/modules/content_panel/content_panel.css:
```diff
@@ -0,0 +1,26 @@
+  }
+
+  .panel-figure{
+    flex: 0 0 auto;
```
Si!!! Buen uso de `flex-basis: auto`!! 💯

misscs reviewed (changes requested):
Solamente una cosa para arreglar!! 🙌

misscs commented:
@Morban15 Puedes terminar este PR primero esta mañana? Gracias!

Morban15 committed:
Cambiando de img a la clase "panel-img" en css

Ya teniamos una clase en html para la imagen, pero se nos
olvidó cambiarlo en el archivo de css.

Morban15 committed:
Ordenando códigos en el archivo de css

Hemos ordenado los códigos en desde mas grande hasta el mas pequeño.

misscs committed:
Merge branch 'master' into content_panel

Conflicts:
	app/assets/styles/site.css
	app/views/programs/index.html
	content/en/programs.yml
	content/es/programas.yml
